ios/HathorMobile.xcodeproj/project.pbxproj

```diff
@@ -512,7 +512,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = HathorMobile/HathorMobile.entitlements;
-				CURRENT_PROJECT_VERSION = 0.3.0;
+				CURRENT_PROJECT_VERSION = 0.4.0;
```

Comment:
When I started to work on the mobile we didn't change the `CURRENT_PROJECT_VERSION`. I'm wondering when it started to be changed and why?